pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -579,7 +579,7 @@ cdef class _Timestamp(ABCTimestamp):
         if freq:
             kwds = freq.kwds
             month_kw = kwds.get("startingMonth", kwds.get("month", 12))
-            freqstr = freq.freqstr
+            freqstr = freq.name
```

Comment:
variable name

Reply:
sorry, it's my mistake. I renamed the variable `freqstr` to `freq_name`